<?xml version="1.0" encoding="UTF-8"?>
<testsuites name="Mocha Tests" time="26.934" tests="1" failures="1">
  <testsuite name="Root Suite" timestamp="2018-04-14T09:34:14" tests="0" failures="0" time="0">
  </testsuite>
  <testsuite name="Admin Ui Testing" timestamp="2018-04-14T09:34:14" tests="1" file="C:\Users\shadab.khan\Documents\GitHub\nodetest\test\demo.js" failures="1" time="26.934">
    <testcase name="Admin Ui Testing Test For List elements for partner" time="26.934" classname="Test For List elements for partner">
      <failure><![CDATA[AssertionError: expected 'Organization Name' to not equal 'Organization Name'
    at Context.<anonymous> (test\demo.js:94:20)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)]]></failure>
    </testcase>
  </testsuite>
</testsuites>